frontend/src/components/placementVisuals/Placement.tsx

```diff
@@ -162,6 +162,18 @@ function Placement({ stepID, sampleIDs }: PlacementProps) {
         }
     }, [activeDestinationContainer, dispatch])
 
+    // useEffect(() => {
+    //     (async () => {
+    //         const store = (await import("../../store")).default
+    //         const placementModule = await import("../../modules/placement/classes")
+    //         console.info({
+    //             getPlacementClass: (parentContainer: ParentContainerIdentifier | undefined = undefined) => new placementModule.PlacementClass(store.getState().placement, parentContainer),
```

Comment:
Remove please (do you want to have comments with my debug stuff in there too ? Imagine if we were 3 putting those all over the place ... ;)